lib/components/WeatherCards.js

```diff
@@ -1,30 +1,32 @@
 import React from 'react';
 
-const WeatherCards = ({ weather }) => {
-  if(!weather.length){
+const WeatherCards = ({ weatherText, weatherSimple, city }) => {
+  if(!weatherText.forecastday) {
```

Comment:
destructuring out weatherText forecast and weatherSimple forecast. Useful for daily weather summary